webapp/src/webapp/subs.cljs

```diff
@@ -358,3 +359,28 @@
  :reports->session
  (fn [db _]
    (:reports->session db)))
+
+(re-frame/reg-sub
+ :reports->redact-data-by-date
+ (fn [db _]
+   (:reports->redact-data-by-date db)))
```

Comment:
it's a minor, but still helps readability to have the `redact` in its verbal time right as `redacted` across this subs and events at the `reports/cljs`. Saying `redact` means an action, versus what this demonstrate, which is redacted information